.maestro/run_ui_tests.sh

```diff
@@ -38,7 +38,7 @@ run_flow() {
 	echo "⏲️ Starting flow $( basename $flow)"
 
 	export MAESTRO_DRIVER_STARTUP_TIMEOUT=60000
-	maestro --udid=$device_uuid test $flow
+	maestro --udid=$device_uuid test -e ONBOARDING_COMPLETED=true $flow
```

Comment:
We can probably enhance the script to add optional env variables to pass when we run it